pkg/host/syscalls_linux.go

```diff
@@ -235,6 +235,8 @@ func isSupportedSyzkall(c *prog.Syscall, target *prog.Target, sandbox string) (b
 			return false, fmt.Sprintf("sys_%v is not present in the target", ioUringSyscallName)
 		}
 		return isSupportedSyscall(ioUringSyscall, target)
+	case "syz_btf_id_by_name":
+		return true, ""
```

Comment:
I think I need to change this to something like "return true only if `/sys/kernel/btf/vmlinux`" exists. Is this correct?

Reply:
Yes, I think it's a good condition to enable/disable this call.